packages/now-build-utils/test/unit.test.js

```diff
@@ -453,15 +453,15 @@ it('Test `detectBuilders`', async () => {
     };
     const functions = {
       'api/users/*.ts': {
-        runtime: 'my-custom-runtime-package',
+        runtime: 'my-custom-runtime-package@1.0.0',
       },
       'api/teams/members.ts': {
         memory: 128,
         maxDuration: 10,
       },
       'package.json': {
         memory: 3008,
-        runtime: '@now/next@canary',
+        runtime: '@now/next@1.0.0-canary',
```

Comment:
```suggestion
        runtime: '@now/next@1.0.0-canary.12',
```

Let's make this match how we do versioning just to be sure